src/main/java/wolox/training/utils/Utils.java

```diff
@@ -0,0 +1,35 @@
+package wolox.training.utils;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.time.LocalDate;
+
+public class Utils {
```

Comment:
Maybe we can use some more specific name for this class, for example MessagesUtils or something like that
Because if we use a class called Utils we will tend to add a lot of methods that are no related between each other

(If we can't find a class name to represent every method here, maybe we should divide this class in two or more utils classes)

Reply:
TRUE!